### PR TITLE
Initial hearing-api merge

### PR DESCRIPTION
**Pull request by srochani**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by sanjayparekh-hmcts — DISMISSED** [review]

**Review by adusumillipraveen — DISMISSED** [review]

**Comment by srochani** [comment]
closing and reopening

**Comment by srochani** [comment]
closing the PR

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -12,100 +12,100 @@ def dmnBranch = "master"
 def waStandaloneBranch = "master"
 
 def secrets = [
-    'sscs-${env}': [
```
Comment (sanjayparekh-hmcts) [inline Jenkinsfile_CNP:15]: All these look like just a whitespace change - can you confirm and if not needed, revert to master please

**Review by SomeshRA — APPROVED** [review]

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

**Comment by srochani** [comment]
closing and opening new PR

**Pull request closed without merging**